From 9c5366fb3757acc47257be70b617eab4c0b3db68 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Thu, 29 Apr 2021 15:35:04 +0000
Subject: [PATCH] Fix build with gcc-11

* fixes:
In file included from anbox/3.0+gitAUTOINC+9de4e87cdd-r0/git/src/anbox/input/manager.cpp:18:
anbox/3.0+gitAUTOINC+9de4e87cdd-r0/git/src/anbox/input/manager.h:38:8: error: 'uint32_t' in namespace 'std' does not name a type; did you mean 'wint_t'?
   38 |   std::uint32_t next_id();
      |        ^~~~~~~~
      |        wint_t

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 src/anbox/input/manager.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/anbox/input/manager.h b/src/anbox/input/manager.h
index 22f1406..f9cbe91 100644
--- a/src/anbox/input/manager.h
+++ b/src/anbox/input/manager.h
@@ -20,6 +20,7 @@
 
 #include <map>
 #include <memory>
+#include <cstdint>
 
 namespace anbox{
   class Runtime;
